README.md

```diff
@@ -63,6 +64,30 @@ yarn run dev
 
 In this project we are using `yarn` because `netlify` builds our site with `yarn`.
 
+### Folder style guide
+
+W> For new tech topics, you will need to create a new folder.
+W> The URL of an article is created from the folder title.
+W> Folder names should not contain the following characters:
+W>  
+W> < (less than) 
+W> > (greater than)
+W> : (colon)
+W> " (double quote)
+W> / (forward slash)
+W> \ (backslash)
+W> | (vertical bar or pipe)
+W> ? (question mark)
+W> * (asterisk)
+W>   (whitespace)
+
+>[What characters are forbidden in Windows and Linux folder names?](https://stackoverflow.com/questions/1976007/what-characters-are-forbidden-in-windows-and-linux-directory-names#1976050)
+><cite>Christopher Oezbek</cite>
+
+T> The dash character `-` is the ONLY accepted character in folder names other than letters and numbers.
```

Comment:
I think we can shorten this entire section to something like this:
"The folder name is used in the URL, so only use dashes, numbers, and lowercase letters for it."